added release notes up to 10.17.100.0 (#163)
SoftwareAG/c8y-releasenotes: content/release-10-18-0/applications-10-18-0.md

title: Application enablement
layout: bundle
---

<!--10.17.1.0 - 10.17.100.0-->

### Improvements

<div><table ><colgroup>
<col style="width: 15%;"><col style="width: 55%;"><col style="width: 15%;"><col style="width: 15%;"></colgroup><thead><tr>
<th>
Component</th>
<th>
Description</th>
<th>
Issue</th>
<th>
Fix Version</th>
</tr>
</thead><tbody>

<tr>
<td>Cockpit</td>
<td>The "Rotation" widget has been migrated to Angular.</td>
<td>MTM-50847</td>
<td>10.17.68.0</td>
</tr>

<tr>
<td>Cockpit</td>
<td>Widgets that use the global date context are now marked with an icon and a corresponding tooltip.</td>
<td>MTM-51225</td>
<td>10.17.68.0</td>
</tr>

<tr>
<td>Cockpit</td>
<td>It is now possible to use the global time range in "Data point table" widgets.</td>
<td>MTM-50924</td>
<td>10.17.68.0</td>
</tr>

<tr>
<td>Cockpit</td>
<td>It is now possible to use the global time range in "Event list" widgets.</td>
<td>MTM-50925</td>
<td>10.17.67.0</td>
</tr>

<tr>
<td>Cockpit</td>
<td>It is now possible to use the global time range in "Data point graph" widgets.</td>
<td>MTM-48198</td>
<td>10.17.11.0</td>
</tr>

<tr>
<td>Web SDK</td>

> <td>It is now possible to open a blueprint directly after the installation has been finished.</td>

Reviewer: ```suggestion
<td>It is now possible to open a blueprint directly after the installation has finished.</td>

```

<td>MTM-50637</td>
<td>10.17.47.0</td>
</tr>

<tr>
<td>Web SDK</td>
<td>The <code>HOOK_COMPONENTS</code> now allows to lazily load the components used, for example, within widgets by utilizing the <code>loadComponent</code> and <code>loadConfigComponent</code> attributes of the <code>DynamicComponentDefinition</code> type (see <a href="https://resources.cumulocity.com/documentation/websdk/1016.0.110/ngx-components/interfaces/LazyDynamicComponents.html" class="no-ajaxy">https://resources.cumulocity.com/documentation/websdk/1016.0.110/ngx-components/interfaces/LazyDynamicComponents.html</a>).</td>
<td>MTM-51161</td>
<td>10.17.27.0</td>
</tr>

</tbody></table></div>

### Fixes

<div><table ><colgroup>
<col style="width: 15%;"><col style="width: 55%;"><col style="width: 15%;"><col style="width: 15%;"></colgroup><thead><tr>
<th>
Component</th>
<th>
Description</th>
<th>
Issue</th>
<th>
Fix Version</th>
</tr>
</thead><tbody>

<tr>
<td>Cockpit</td>

> <td>An issue has been fixed that caused angularJS widgets to not show up in the "Add widget" dialog.</td>

Reviewer: ```suggestion
<td>An issue has been fixed that caused AngularJS widgets to not show up in the "Add widget" dialog.</td>

```

<td>MTM-51482</td>
<td>10.17.67.0</td>
</tr>

<tr>
<td>Web SDK</td>

> <td>Opening the widget configuration for a microfrontend plugin multiple times does no longer cause performance issues in the UI.</td>

Reviewer: ```suggestion
<td>Opening the widget configuration for a micro frontend plugin multiple times does no longer cause performance issues in the UI.</td>

```

Author: Microfrontend is the way we spell it throughout the documentation. Both spellings seem to be correct.

Reviewer: no results for "microfrontend", 21 results for "micro frontend" when i checked

Author: Where did you check?

Author: You checked in c8y-docs it seems. You are right, we spell it different there. In the release notes it is spelled in one word, I will change it in here.

<td>MTM-50306</td>
<td>10.17.53.0</td>
</tr>

<tr>
<td>DTM</td>

> <td>If the DTM asset custom property is declared as required and complex, all of it's subproperties are required too.</td>

Reviewer: ```suggestion
<td>If the DTM asset custom property is declared as required and complex, all of its subproperties are required too.</td>

```
